tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -654,7 +654,7 @@ EOF
 			#   	OPA should be built from the cached source code instead of downloading
 			#   	this binary.
 			#
-			opa_bin_url="$(get_package_version_from_kata_yaml externals.open-policy-agent.meta.binary)"
+			opa_bin_url="$(get_package_version_from_kata_yaml externals.open-policy-agent.architecture.${ARCH}.binary)"
```

Comment:
In this part here, we'll actually need to point to the jenkins job URL, which we'll create.

Then we download the version from there, see whether it matches with the one coming from the versions.yaml file, and then decide whether we need to build it or not.

I'd leave this as a second PR so we can easily get the other parts merged first, and then we just do the switch here.